subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -243,6 +244,7 @@ int nrf_cloud_fota_pending_job_validate(enum nrf_cloud_fota_type * const fota_ty
 static void fota_reboot(void)
 {
 	LOG_INF("Rebooting to complete FOTA update...");
+	LOG_PANIC();
```

Comment:
Good catch -- I would add a comment here that we do this to make sure any deferred logs get flushed (unless there is another reason you're adding this!)